src/app/rosetta/README.md

```diff
@@ -6,13 +6,15 @@ Implementation of the [Rosetta API](https://www.rosetta-api.org/) for Mina.
 
 Checkout the "master" branch of the mina repository, ensure your Docker configuration has a large amount of RAM (at least 12GB, recommended 16GB) and then run the following:
 
+### Release v1.3.0 (Mainnet and Devnet)
```

Comment:
not a blocker but on documentation latest stable release is 1.3.1.2

Reply:
Good catch @simisimis. I have updated the rest of the Readme accordingly.